### ISSUE TITLE
For Disabled people pre update Super mouse sensitive was best

### ISSUE THREAD

**Issue by reaven114** [uberhalit/EldenRingFpsUnlockAndMore#50]
Is it possible to revert the mouse sensitivity to the pre-update settings? or increase the setting past 10?

**Comment by uberhalit** [comment]
Hello, 
although this might be possible I'd suggest trying to increase DPI on your mouse instead. Modern gaming mice go up to 32000 dpi which would let you do a 360° turn in less than 2cm.

**Comment by reaven114** [comment]
Like I get close to that to with my mouse max 25600 dpi but at least for me that's just not enough, which was sad because like I had mentioned the pre-update was perfect, but I know it might not be possible. I actually wonder if maybe they added mouse smoothing in the update, in my experience turning it off fixes most of my problems.

**Comment by DubbleClick** [comment]
windows settings max +25600 dpi will make your mouse move an entire 4k screen in 2 millimeters

**Comment by reaven114** [comment]
> windows settings max +25600 dpi will make your mouse move an entire 4k screen in 2 millimeters

you're right if you're not gaming and just doing normal computer stuff, but some games like Elden Ring and others override or set max level that pretty much negates dpi settings.

**Comment by DubbleClick** [comment]
I've just tried to increase my dpi and at least in windowed fullscreen, the game starts spinning incredibly fast at the slightest movement.

**Comment by reaven114** [comment]
> I've just tried to increase my dpi and at least in windowed fullscreen, the game starts spinning incredibly fast at the slightest movement.

Please don't take this the wrong way, but that may be true for you but trust me when I say it's just not that way for everyone depending on the disability as insane as it sounds it's too slow.

Now for me do you think it's my game settings that could be messing with me?

**Comment by DubbleClick** [comment]
> Now for me do you think it's my game settings that could be messing with me?

Yeah, I think it's your game doing weird things. Either because of fullscreen, or because... elden ring. I've had such strange issues with this game that seemingly came out of nowhere and had no fixed, I reinstalled windows clean and everything is working great.. for now.

**Comment by reaven114** [comment]
I'll give that a try.

**Comment by uberhalit** [comment]
If you can't make it work I'll look into the sensitivity scaling of the game and see if I can offer a specialized tool/CheatEngine Table to adjust it on the fly.

**Comment by reaven114** [comment]
> If you can't make it work I'll look into the sensitivity scaling of the game and see if I can offer a specialized tool/CheatEngine Table to adjust it on the fly.

if possible u should I'm sure others would appreciate it. :)
